pkg/profilestore/profilestore.go

```diff
@@ -65,6 +66,10 @@ func (s *ProfileStore) WriteRaw(ctx context.Context, r *profilestorepb.WriteRawR
 	for _, series := range r.Series {
 		ls := make(labels.Labels, 0, len(series.Labels.Labels))
 		for _, l := range series.Labels.Labels {
+			if err := model.LabelName(l.Name).IsValid(); !err {
```

Comment:
```suggestion
			if valid := model.LabelName(l.Name).IsValid(); !valid {
```

This is more idiomatic. `err` usually reserved for a type conforms `error` interface.